### ISSUE TITLE
Add a flag to output html instead of (or besides) pdf.

### ISSUE THREAD

**Issue by tplk** [simonhaenisch/md-to-pdf#30]
In my case, I am writing my CV in Markdown and it'd be helpful to have both pdf and html versions to send pdf via email and have an html version on my website.

**Comment by simonhaenisch** [comment]
@tplk I'm planning to rewrite a few things related to how input and output works soonish, so I'll keep this in mind... should be straight-forward to implement.

Thanks for the idea!

**Comment by tplk** [comment]
@simonhaenisch I've read through the code yesterday, seems quite easy to implement even without refactoring. Thanks!

**Comment by simonhaenisch** [comment]
Yeah, if you want to, you can make a PR...

- add a flag `--as-html` (and config option `as_html`)
- move `lib/get-pdf-file-path.js` to `lib/get-file-path.js`, then change the function to take extension as an argument (so either `.pdf` or `.html` can be passed)
- in a new file `lib/write-html.js` implement writing the file using async `writeFile` from `fs` module

https://github.com/simonhaenisch/md-to-pdf/blob/a4a2d2911df53e15d7987438290fdb2a0c908c06/lib/md-to-pdf.js#L66-L74

Change this to sth like

```js
const output = config.as_html
  ? await writeHtml(mdFile, html, config)
  : await writePdf(mdFile, html, { ...config, port });

if (!output.filename) {
  throw new Error(`Failed to create ${config.as_html ? 'HTML' : 'PDF'}`);
}

return output;
```

(`writeHtml` will also have to return an object with `filename` key)

**Comment by tplk** [comment]
Yeah, thanks for the directions, I'll probably look into it in the evening (~ in 8 hours).

**Comment by simonhaenisch** [comment]
@tplk I just released [v2.7.0](https://github.com/simonhaenisch/md-to-pdf/releases/tag/v2.7.0) which adds an `--as-html` flag... hope that suits your needs!

**Comment by tplk** [comment]
That's awesome, thanks!
